_search-plugins/knn/approximate-knn.md

```diff
@@ -9,23 +9,26 @@ has_math: true
+Standard k-NN search methods compute similarity using a “brute-force” approach that measures nearest distance between a query and a number of points, which produces exact results. This works well in many applications. However, in the case of extremely large data sets with high dimensionality, this creates a scaling problem that reduces the efficiency of the search. Approximate k-NN search methods can overcome this by employing tools that restructure indexes more efficiently and reduce the dimensionality of searchable vectors. Using this approach requires a sacrifice in accuracy but increases search processing speeds appreciably.
 
-The k-NN plugin builds a native library index of the vectors for each "knn-vector field"/ "Lucene segment" pair during indexing that can be used to efficiently find the k-nearest neighbors to a query vector during search. To learn more about Lucene segments, see the [Apache Lucene documentation](https://lucene.apache.org/core/8_9_0/core/org/apache/lucene/codecs/lucene87/package-summary.html#package.description).
-These native library indices are loaded into native memory during search and managed by a cache. To learn more about
-pre-loading native library indices into memory, refer to the [warmup API]({{site.url}}{{site.baseurl}}/search-plugins/knn/api#warmup-operation). Additionally, you can see what native library indices are already loaded in memory, which you can learn more about in the [stats API section]({{site.url}}{{site.baseurl}}/search-plugins/knn/api#stats).
+The approximate k-NN search methods leveraged by OpenSearch use approximate nearest neighbor (ANN) algorithms from nmslib, faiss, and Lucene libraries to power k-NN search. These search methods employ ANN to improve search latency over large data sets. Of the three search methods the k-NN plugin provides, this method offers the best search scalability for large data sets. This approach is the preferred method when a data set reaches hundreds of thousands of vectors.
 
-Because the native library indices are constructed during indexing, it is not possible to apply a filter on an index
+For details on the algorithms the plugin currently supports, see [k-NN Index documentation]({{site.url}}{{site.baseurl}}/search-plugins/knn/knn-index#method-definitions).
+{: .note}
+
+The k-NN plugin builds a native library index of the vectors for each "knn-vector field"/"Lucene segment" pair during indexing, which can be used to efficiently find the k-nearest neighbors to a query vector during search. To learn more about Lucene segments, see the [Apache Lucene documentation](https://lucene.apache.org/core/8_9_0/core/org/apache/lucene/codecs/lucene87/package-summary.html#package.description).
```

Comment:
I don 't think quotation marks are necessary here.

Reply:
This content may change after technical review. But I agree they're not needed here.